discovery-provider/src/queries/get_trending_tracks.py

```diff
@@ -1,18 +1,40 @@
 import logging  # pylint: disable=C0302
 import datetime
 from dateutil.parser import parse
-from flask.globals import request
 
 from src.utils.db_session import get_db_read_replica
 from src.queries.get_unpopulated_tracks import get_unpopulated_tracks
 from src.queries.query_helpers import populate_track_metadata, \
     get_users_ids, get_users_by_id
 from src.tasks.generate_trending import generate_trending
-from src.utils.redis_cache import extract_key, use_redis_cache
+from src.utils.redis_cache import use_redis_cache
 
 logger = logging.getLogger(__name__)
 
-SCORES_CACHE_DURATION_SEC = 10 * 60
+TRENDING_LIMIT = 100
+
+def make_trending_cache_key(time_range, genre):
+    """Makes a cache key resembling `generated-trending:week:electronic`"""
+    return f"generated-trending:{time_range}:{(genre.lower() if genre else '')}"
+
+def generate_unpopulated_trending(session, genre, time_range):
+    trending_tracks = generate_trending(
```

Comment:
seems like a weird end bracket?

Reply:
Surprised that lint didn't pick this up, but I can adjust